test/e2e/mnist_rayjob_raycluster_test.go

```diff
@@ -402,17 +429,15 @@ func constructRayJob(_ Test, namespace *corev1.Namespace, rayCluster *rayv1.RayC
 		},
 		Spec: rayv1.RayJobSpec{
 			Entrypoint: "python /home/ray/jobs/mnist.py",
-			RuntimeEnvYAML: `
-  pip:
-    - pytorch_lightning==2.4.0
-    - torchmetrics==1.6.0
-    - torchvision==0.20.1
-  env_vars:
-    MNIST_DATASET_URL: "` + GetMnistDatasetURL() + `"
-    PIP_INDEX_URL: "` + GetPipIndexURL() + `"
-    PIP_TRUSTED_HOST: "` + GetPipTrustedHost() + `"
-    ACCELERATOR: "` + accelerator + `"
-`,
+			RuntimeEnvYAML: fmt.Sprintf(`
```

Comment:
It is also possible to generate the yaml from struct - i.e. using https://pkg.go.dev/sigs.k8s.io/yaml#section-readme.
Though, as this yaml handling is relevant just for this one place - it is ok to keep current implementation.